.github/workflows/next-binaries.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Nightly Next Binaries
+on:
+  schedule:
+    - cron: '10 3 * * *'
+  workflow_dispatch:
+    name:
+      description: 'Name of the ouput file'
+      required: false
+      default: ''
+
+# This workflow is mostly a copy of publish binaries
+# This is by choice as the next branch will eventually be merged in the master
+# It makes it easy to remove when the time comes
+jobs:
+  pre_release_bin:
+    name: Build and Publish Binaries
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@next
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12.18.1'
+          cache: 'yarn'
+
+      - name: Configure SSH Key
+        uses: botpress/gh-actions/set_ssh_key@v1
+        with:
+          ssh_key: ${{ secrets.BP_PRO_SSH }}
+
+      - name: Install python3
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+          architecture: 'x64'
+
+      - name: Install Tools
+        run: pip install awscli
+
+      - name: Build Project
+        run: |
+          yarn
+          yarn build --prod
+        env:
+          EDITION: pro
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create Archives
+        run: yarn cmd archive
+
+      - name: Extract Informations
+        id: info
+        uses: botpress/gh-actions/extract_info@v1
```

Comment:
This step does not seem to be used anywhere

Reply:
will remove thanks